src/lib.rs

```diff
@@ -522,46 +531,39 @@ impl Inotify {
     ///
     /// [`Inotify`]: struct.Inotify.html
     /// [`close`]: ../libc/fn.close.html
-    pub fn close(mut self) -> io::Result<()> {
+    pub fn close(self) -> io::Result<()> {
         // `self` will be dropped when this method returns. The `Drop`
         // implementation will attempt to close the file descriptor again,
         // unless this flag here is cleared.
-        self.close_on_drop = false;
+        self.fd.close_on_drop.store(false, Ordering::SeqCst);
```

Comment:
Since `Drop` is no longer implemented for `Inotify`, could you update the comment to note that it refers to the `Drop` implementation of `FdGuard`?